Application/gui_utils.py

```diff
@@ -9,11 +9,11 @@
 from PyQt5.QtCore import *
 from PyQt5.QtGui import *
 from PyQt5.QtWidgets import *
+from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT as NavigationToolbar
```

Comment:
We should separate the import statements into groups and with a comment

More organized

Reply:
Alright